From 834e3a2a2082966dcd04637e2ca35c550d0960e4 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 07:57:11 +0300
Subject: [PATCH 38/50] fdts: s32cc: Add SUSE compatible for SDHC

Add "nxp,s32g2-sdhc" to compatible list to boot SUSE images based
on Linux Kernel 5.17.

SUSE images:
https://download.opensuse.org/repositories/home:/clin:/openSUSE:/Tumbleweed:/S32G2/images/

Issue: ALB-8903
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index a56abaf95..56fbd1e5f 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -732,7 +732,7 @@
 		};
 
 		usdhc0: usdhc@402f0000 {
-			compatible = "nxp,s32cc-usdhc";
+			compatible = "nxp,s32cc-usdhc", "nxp,s32g2-usdhc";
 			reg = <0x0 0x402f0000 0x0 0x1000>;
 			interrupts = <GIC_SPI 36 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&clks S32GEN1_SCMI_CLK_USDHC_MODULE>,
-- 
2.17.1

